juju/juju
# add me-south-1 ec2 region.  add ec2 product family "compute instance (bare metal)"

## Description of change

Add me-south-1 ec2 region.  Add ec2 product family "compute instance (bare metal)"

## QA steps

1. juju regions aws
2. Verify endpoints here: https://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region
3. attempt bootstrap or charm deployment with i3.metal instance type.
juju bootstrap aws/us-east-1 --constraints instance-type="i3.metal"

## Bug reference

https://bugs.launchpad.net/juju/+bug/1839496
